>proxy temp/proxyProxy set up at localhost:29192
>source '/afs/andrew.cmu.edu/usr18/sarthak3/proxylab-m18-sarthakjain27/tests/A08-multiple-fetch.cmd'># Test ability to handle multiple fetches
>serve s1 s2 s3
Server s1 running at localhost:29193
Server s2 running at localhost:29194
Server s3 running at localhost:29195
>generate random-text1.txt 2K 
>generate random-text2.txt 4K 
>generate random-text3.txt 6K
>generate random-text4.txt 8K
>generate random-text5.txt 10K
>generate random-text6.txt 12K
>fetch f1 random-text1.txt s1
Client: Fetching '/random-text1.txt' from localhost:29193
>fetch f2 random-text2.txt s2
Client: Fetching '/random-text2.txt' from localhost:29194
>fetch f3 random-text3.txt s3
Client: Fetching '/random-text3.txt' from localhost:29195
>fetch f4 random-text4.txt s3
Client: Fetching '/random-text4.txt' from localhost:29195
>fetch f5 random-text5.txt s2
Client: Fetching '/random-text5.txt' from localhost:29194
>fetch f6 random-text6.txt s1
Client: Fetching '/random-text6.txt' from localhost:29193
>delay 400
>check f1
ERROR: Request f1 generated status 'error'.  Expecting 'ok' (Got empty response for URL request http://localhost:29193/random-text1.txt)
>check f2
ERROR: Request f2 generated status 'error'.  Expecting 'ok' (Couldn't connect to localhost:29192 ([Errno 111] Connection refused))
>check f3
ERROR: Request f3 generated status 'error'.  Expecting 'ok' (Couldn't connect to localhost:29192 ([Errno 111] Connection refused))
>check f4
ERROR: Request f4 generated status 'error'.  Expecting 'ok' (Couldn't connect to localhost:29192 ([Errno 111] Connection refused))
>check f5
ERROR: Request f5 generated status 'error'.  Expecting 'ok' (Couldn't connect to localhost:29192 ([Errno 111] Connection refused))
Too many errors.  Exiting
ERROR COUNT = 5
